packages/components/psammead-script-switch/src/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { C_WHITE, C_EBON } from '@bbc/psammead-styles/colours';
+import { string, shape, node } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const StyledLink = styled.a`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => service && getSansRegular(service)}
+
+  display: inline-block;
+  border: 0.0625rem solid ${C_WHITE};
+  color: ${C_WHITE};
+  text-decoration: none;
+  padding: 0 1rem;
+  height: 3rem;
+
+  &:hover {
+    border-width: 0.25rem;
+    line-height: 2.62rem;
+    padding: 0 0.81rem;
+  }
+  &:focus {
+    background-color: ${C_WHITE};
+    color: ${C_EBON};
+  }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    line-height: 3rem;
```

Comment:
Sorry, removed the comment by mistake. In that case, should we put `padding: 0 1rem;` and `height: 3rem;` in here too?

Otherwise the `@media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN})` is overriding them until 600px.